variables.tf

```diff
@@ -207,7 +207,7 @@ variable "worker_create_security_group" {
 variable "permissions_boundary" {
   description = "If provided, all IAM roles will be created with this permissions boundary attached."
   type        = string
-  default     = ""
+  default     = null
```

Comment:
This won't break `aws_iam_role` ? Because, it's a null string (`"null"`) and not `null` value.

Reply:
it's not null string, 0.12 introduces the null value for conditionally omitting a value.

Reply:
https://www.hashicorp.com/blog/terraform-0-12-conditional-operator-improvements

Reply:
So it's probably terraform-doc is writing it as string in docs

> | permissions\_boundary | If provided, all IAM roles will be created with this permissions boundary attached. | string | `"null"` | no |

Reply:
There is no native support for Terraform 0.12 in terraform-docs. See https://github.com/segmentio/terraform-docs/issues/62

Reply:
I know. I was thinking about the [awk version](https://github.com/antonbabenko/pre-commit-terraform/blob/master/terraform_docs.sh) in the pre-commit hooks.

Reply:
Maybe here https://github.com/antonbabenko/pre-commit-terraform/blob/master/terraform_docs.sh#L138

Reply:
Never mind, all types are "quoted" by terraform-docs.sh. I using this for a long time now, but I didn't notice that until today :s